pkg/yamlmeta/node.go

```diff
@@ -57,8 +58,8 @@ func (m *Map) SetValue(val interface{}) error {
 }
 
 func (mi *MapItem) SetValue(val interface{}) error {
```

Comment:
what about AddValue(...)?

Reply:
should we implement SetValue as ResetValue + AddValue?

Reply:
🤦🏼 , 🙏🏼 , 👍🏼 .

Reply:
> what about AddValue(...)?

re:🤦🏼  Yup!

> should we implement SetValue as ResetValue + AddValue?

Will seriously consider this (for this exact reason).